EssentialsDiscord/src/config.yml

```diff
@@ -0,0 +1,31 @@
+#version: ${full.version}
+#############################################################
+# +-------------------------------------------------------+ #
+# |                  EssentialsX Discord                  | #
+# +-------------------------------------------------------+ #
+#############################################################
+
+# This config file only works when the EssentialsXDiscord jar is installed.
+
+# You need to create a bot user in order to connect your server to Discord.
+# You can find instructions on this here: #TODO write up docs
+
+# The token for your bot from the Discord Developers site.
+token: "INSERT-TOKEN-HERE"
+
+# The ID of your server.
+guild: 0
+
+# The channels that will be used for messaging, defaulting to the primary channel.
+channels:
+  primary:
+    id: 0
+
+# Determine which channels certain messages will go to.
+messages:
+  chat: primary
+
+# Status message for the bot to display. The activity must be playing or listening.
```

Comment:
What is about "watching"?
Also, why not make this optional? You could just not set the activity when it isn't given (afaik can you just pass `null` in the setActivity method for no activity)

Reply:
If I recall correctly the only officially supported activity types were "playing" or "listening". The API has more, and surely you can use them if you know about it, however I opted not to document it for the time being.

Good suggestion about making this optional. That should be easily doable.